3.7.0-full/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# SonarQube 7.8 image with bundled sonar-scala 7.6.0 (https://github.com/mwz/sonar-scala).
+
+FROM sonarqube:7.8-community
```

Comment:
Are you sure you want to keep a separate folder for every old version you had, despite it's outdated? 
Would it be better just to change one Dockerfile upgrading some versions?